express.js

```diff
@@ -1,42 +1,76 @@
-const path            = require('path');
-const fs              = require('fs')
-const express         = require('express')
-const qsets           = path.join(__dirname, 'qsets');
-const yaml            = require('yamljs');
-const { execSync }    = require('child_process');
-const waitUntil       = require('wait-until-promise').default
-const hoganExpress    = require('hogan-express')
-const uuid            = require('uuid')
-const sharp           = require('sharp')
-
-var webPackMiddleware = false;
-var hasCompiled = false;
+const path            	= require('path');
+const fs              	= require('fs');
+const express         	= require('express')
+const qsets           	= path.join(__dirname, 'qsets');
+const yaml            	= require('yamljs');
+const { execSync }    	= require('child_process');
+const waitUntil       	= require('wait-until-promise').default
+const { v4: uuidv4 }    = require('uuid')
+const sharp           	= require('sharp')
+const util				= require('util');
+const cors 				= require('cors')
+const hbs				= require('hbs');
+
+// common paths used here
+const srcPath 				= path.join(process.cwd(), 'src') + path.sep
+const outputPath 			= path.join(process.cwd(), 'build') + path.sep
+
+// Webpack middleware setup
+const webpack 							= require('webpack');
+const webpackDevMiddleware 	= require('webpack-dev-middleware');
```

Comment:
This doesn't really matter, but stick to spaces for inline alignment.

Would prefer to remove all the extraneous white space, personally.